### PR TITLE
Mssql setup

### PR DESCRIPTION
**Pull request by katbow**
Sets up Mssql & checks all queries #852 
Changes queries for clinician #851 
Updates all tests from changes #852

### CODE REVIEW

**Comment by katbow** [comment]
TODO: update heroku variables before merging in

**Comment by nelsonic** [comment]
@katbow code looks good. but pending: https://github.com/healthlocker/healthlocker/issues/856

File: config/dev.exs [hunk 1]
```diff
@@ -43,11 +43,7 @@ config :healthlocker, Healthlocker.Repo,
...
 
 config :healthlocker, Healthlocker.ReadOnlyRepo,
-  adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
-  database: "epjs_dev",
-  hostname: "localhost",
-  pool_size: 10
+  adapter: MssqlEcto,
+  hostname: "localhost"
```
Comment (nelsonic) [inline config/dev.exs]: @katbow does this configuration work on the Azure Staging Server?

**Comment by nelsonic** [comment]
Steps taken to review PR:
```
ssh hladmin@51.140.86.5
sudo -i
source /home/hladmin/.profile
cd /home/hladmin/hl-app
mix phoenix.server
```
Result:
image

**Comment by nelsonic** [comment]
No longer deploying to heroku. 👍

**Pull request merged**